flips/20210113-quorum-certificate.md

```diff
@@ -0,0 +1,153 @@
+# Epoch Collector Cluster Quorum Certificate Smart Contract
+
+| Status        | (Proposed)       |
+:-------------- |:---------------------------------------------------- |
+| **FLIP #**    | [313](https://github.com/onflow/flow/pull/NNN) (update when you have PR #)|
+| **Feature PR** | [QC Contract Draft](https://github.com/onflow/flow-core-contracts/pull/66) |
+| **Author(s)** | Josh Hannan (joshua.hannan@dapperlabs.com) |
+| **Sponsor**   | Josh Hannan (joshua.hannan@dapperlabs.com)            |
+| **Updated**   | 2021-01-13                                           |
+
+## Objective
+
+Enable quorum certificate generation for bootstrapping the HotStuff consensus algorithm
+in each collector cluster. The voting mechanism will be implemented in a Cadence smart contract.
+
+## Motivation
+
+When epochs are fully implemented on Flow, every epoch organizes all the confirmed collector
+nodes into new clusters. Each cluster has to bootstrap a new HotStuff consensus algorithm
+by gathering votes from a quorum of its collector nodes. (~2/3 of the nodes)
+
+The generating of votes is left to the node software, but the smart contract will act
+as a record of votes for the clusters and an arbiter for which nodes are eligible to vote.
+
+## User Benefit
+
+The benefits of implementing this as a smart contract are:
+* Easier auditability
+* Easier upgradability without having to perform a fork
+* Integration with other epoch-related smart contracts
```

Comment:
👍 When we have it we should link out the Epoch Preparation Protocol doc as well -- a lot of the reasoning for this contract falls out of the general reasoning for mediating epoch transitions via smart contract